pkg/analysis/enums/testdata/src/a/a.go

```diff
@@ -0,0 +1,122 @@
+package a
+
+// Valid enum type with proper marker
+// +kubebuilder:validation:Enum
+type Phase string
+
+const (
+	PhasePending   Phase = "Pending"   // Valid PascalCase
+	PhaseRunning   Phase = "Running"   // Valid PascalCase
+	PhaseSucceeded Phase = "Succeeded" // Valid PascalCase
+	PhaseFailed    Phase = "Failed"    // Valid PascalCase
+)
+
+// Alternative marker format
+// +k8s:enum
+type State string
+
+const (
+	StateActive   State = "Active"   // Valid PascalCase
+	StateInactive State = "Inactive" // Valid PascalCase
+)
+
+// Invalid: Missing +enum marker
+// This type doesn't have +enum marker, so it will be flagged when used in fields
```

Comment:
What triggers this to be flag? A string type alias with some constants assigned?